syntax = "proto3";

message NetConfig{

    string  MqttServerURL  = 1;         
    string  MqttClientName = 2;        
	string  MqttName = 3;			
	string  MqttPassword = 4;		

    string ClientHost      = 5;       
    int32  TCPClientPort   = 6;        

    string ServerHost      = 7;        
    int32  TCPServerPort   = 8;       

}



